src/script/components/AppLock.test.tsx

```diff
@@ -42,21 +44,28 @@ type WriteableConfig = Writable<Configuration>;
 
 const writeableConfig: WriteableConfig = Config.getConfig();
 
-describe('AppLockViewModel', () => {
-  const getAppLock = () =>
-    new AppLockViewModel(
-      ({
+class AppLockPage extends TestPage<AppLockProps> {
+  constructor(props?: AppLockProps) {
+    super(AppLock.AppLock, props);
+    act(() => AppLock.init(props.clientRepository, props.selfUser));
+  }
+}
+describe('AppLock', () => {
+  const initAppLock = () =>
+    new AppLockPage({
+      clientRepository: ({
         clientService: ({
           deleteClient: (clientId: string, password: string) => Promise.resolve(),
         } as unknown) as ClientService,
         currentClient: ko.observable({id: 'clientId'}),
       } as unknown) as ClientRepository,
-      ko.observable(({id: 'userID'} as unknown) as User),
-    );
+      selfUser: ({id: 'userID'} as unknown) as User,
+    });
+
   const originalFeature = writeableConfig.FEATURE;
   beforeEach(() => {
     writeableConfig.FEATURE = {...writeableConfig.FEATURE};
-    document.body.innerHTML = '<div id="app"></div><div id="applock"></div>';
+    document.body.innerHTML = '<div id="wire-main"><div id="app"></div><div id="applock"></div></div>';
```

Comment:
Should not be necessary. Normally we render the whole component (without knockout binding)